src/Api/Response/Parser/FiltersParser.php

```diff
@@ -107,45 +105,49 @@ public function parseForWidgets($data): array
     /**
      * @param string $filterType
      * @param array $filterItem
-     * @param \SimpleXMLElement $data
+     * @param array $data
      * @param int $index
      * @return void
      */
-    public function parseFilterItem($filterType, &$filterItem, $data, $index)
+    public function parseFilterItem($filterType, &$filterItem, array $data, $index)
     {
         if (!empty($data)) {
             $filterItem['items'] = [];
-            $filterItem['name'] = $data->name->__toString();
-            $filterItem['position'] = $index;
-            $filterItem['count'] = $data->frequency->__toString();
+            $filterItem['name'] = $data['value'];
+            $filterItem['position'] = "item";
```

Comment:
@TobiasGraml11 why I named it 'item' everywhere is because in the develop branch every filter item basically has 'item' in it as the position
![image](https://github.com/findologic/plugin-plentymarkets-ceres-api/assets/83945230/e5311873-2917-409e-ad64-185ac56b7fdd)

Reply:
Does the `$index` variable contain "item"? If so, I think it still makes sense to keep it like it is and don't hardcode it there.

Reply:
No, it contains numbers. Guess in xml it was always 'item' cause the xml is something like this:
`<items>
   <item>
      <id>....
    </item>
 </items>`

Reply:
Interesting, please still stick with `$filterItem['position'] = $index;`. I don't think it's used anywhere, but having the numeric values makes more sense.